packages/tabs/src/vaadin-tabs.js

```diff
@@ -236,12 +236,102 @@ class Tabs extends ResizeMixin(ElementMixin(ListMixin(ThemableMixin(PolymerEleme
 
   /** @private */
   _scrollForward() {
-    this._scroll(-this.__direction * this._scrollOffset);
+    const forwardButtonWidth = this._getNavigationButtonVisibleWidth('forward-button');
+    const backButtonWidth = this._getNavigationButtonVisibleWidth('back-button');
+    const scrollerElementBoundingClientRect = this._scrollerElement.getBoundingClientRect();
+    let itemToScrollTo;
+    for (let i = this.items.length - 1; i >= 0; i--) {
+      const isItemVisible = this._isItemVisible(
+        this.items[i],
+        true,
+        forwardButtonWidth,
+        backButtonWidth,
+        scrollerElementBoundingClientRect,
+      );
+      if (isItemVisible) {
+        itemToScrollTo = this.items[i];
+        break;
+      }
+    }
+    const itemBoundingClientRect = itemToScrollTo.getBoundingClientRect();
+    const transitionCompensation = 20;
```

Comment:
```suggestion
    const overflowIndicatorCompensation = 20;
```
Could also add a comment to clarify why it's there and why it's just hard-coded.

Reply:
Added a comment and updated the naming.